package.json

```diff
@@ -1592,6 +1595,306 @@
                         "type": "string",
                         "description": "A path to a file where all brightscript console output will be written. If null or empty, file logging will be disabled.",
                         "scope": "resource"
+                    },
+                    "brightscript.debug.host": {
```

Comment:
All of these need to have `"scope": "resource"` added to them. This should happen in the sync-launch-settings.ts script.

Reply:
Will do ...

Reply:
changes made @TwitchBronBron thanks

Reply:
@philanderson888 The `"scope": "resource"` seems to still be missing in these.

Reply:
Ah yes... That will be because I did not re-run the script yet!!!

Reply:
OK @TwitchBronBron I have now re-run the script.  All of the 'brightscript.debug.' entries now contain "scope":"resource" in them, populated from the script.